packages/react-ui-core/src/Form/RadioGroup.js

```diff
@@ -75,14 +75,15 @@ export default class RadioGroup extends Component {
 
   renderRadioButton(index, fieldProps) {
     const props = pick(this.props, ['name', 'hideInputElement', 'orientation'])
+
     return (
       <RadioButton
-        data-tid="radiogroup-radiobutton"
```

Comment:
I'm not sure why you removed this.

Reply:
I think I accidentally removed it ?

Reply:
Ohh ok I removed this to add data-tid dynamically, but seems like I need to revert this back.

Reply:
You know, I think maybe removing this is fine as long as there is one on the RadioButton already.

Reply:
We dont have that in RadioButton Field, I think it would be better to have it here then.